.github/CODEOWNERS

```diff
@@ -126,6 +126,13 @@
 /code/modules/capture_the_flag/ @NamelessFairy
 /_maps/map_files/CTF/ @NamelessFairy
 
+# Pickle-Coding
+
+/code/__DEFINES/atmospherics/ @Pickle-Coding
+/code/modules/atmospherics/ @Pickle-Coding
+/code/modules/power/ @Pickle-Coding
+/code/__DEFINES/reactions.dm @Pickle-Coding
```

Comment:
This should be alphabetized. Defines come before everything else
yes I will be pranking ryll for not following the same pattern